src/browser-lib/hubspot/forms/hubspotSubmitForm.test.ts

```diff
@@ -130,13 +154,24 @@ describe("hubspotSubmitForm", () => {
 
       expect(reportError).not.toHaveBeenCalled();
     });
+
     it("should report error if fallback fails", async () => {
-      server.use(fallbackForm400UnknownError);
+      const fakeFetch = getFakeFetch([
+        invalidEmailFailure,
+        fallbackHubspotErrorFailure,
+      ]);
+      global.fetch = fakeFetch.asFetch;
+
       try {
         await hubspotSubmitForm({ hubspotFormId, payload });
       } catch (error) {
         //
       }
+
+      // Get the mock response from the fallback form.
+      const fakeFetchResponse = fakeFetch.getResult(1) as ResponsePromise;
```

Comment:
@mantagen maybe these values should be taken from the response matcher config, rather than the return value of the mock fetch function, what do you think?